melodic/distribution.yaml

```
@@ -6149,6 +6149,12 @@ repositories:
       url: https://github.com/ethz-asl/mav_comm.git
       version: master
     status: developed
+  mav_teleop_twist_keyboard:
+    source:
+      type: git
+      url: https://github.com/dobots/mav_teleop_twist_keyboard.git
+      version: master
+    status: maintained  
```

It looks like you just have trailing spaces after the `status: maintained` line.
```suggestion
    status: maintained
```

---

Thanks for the info, I've removed the white spaces.